lasy/profiles/from_array_profile.py

```diff
@@ -68,29 +68,32 @@ def __init__(self, wavelength, pol, array, dim, axes, axes_order=["x", "y", "t"]
             )
         else:  # dim = "rt"
             assert axes_order in [["r", "t"], ["t", "r"]]
+            assert len(array.shape) == 3, "Field array is not 3D."
```

Comment:
With these changes, can we still read FBPIC output successfully? And use lasy as we do for the FBPIC-Wake-T handshake?